css/styles.css

```diff
@@ -45,6 +45,71 @@ main {
     box-shadow: inset 0 0 9px 12px rgba(113,34,250,0.3), 0 0 9px 12px rgba(113,34,250,0.3);
 }
 
+aside div {
+    width: 70px;
+    height: 70px;
+    border: 8px solid white;
+    border-radius: 40px;
+    margin: 0 auto 15px auto;
+    background-size: cover;
+}
+
+.white {
+    border-color: #FFFFFF;
+    background-image: url("../IMG/brushWhite.png");
+    box-shadow: 0 0 5px 5px rgba(255,255,255, 0.3), inset 0 0 5px 5px rgba(255,255,255, 0.3);
```

Comment:
this box shadow and the transition is repeated twice, make into a class?

Reply:
slightly different box shadows for slightly different buttons